source/_includes/head.html

```diff
@@ -1,3 +1,5 @@
+044942
+699311
```

Comment:
Sorry, must've been from accidentally hitting yubikey.  I was doing a bunch of interactive merges, so I thought it had updated some git sha nums, my bad.  they are removed now